### DISCUSSION TITLE
Get `find is not a function` when update tiptap/starter-kit

### DISCUSSION THREAD

**Discussion by loveklmn** [ueberdosis/tiptap#2108] [General Questions] [answered]
This one works well (using the old version of it). https://codesandbox.io/s/tiptap-math-04o0s?file=/src/components/HelloWorld.vue

However, when I updated to the latest version of `starter-kit` and change `default-extensions to `starter-kit`,
something built-in wrong when typing. Try this one. https://codesandbox.io/s/tiptap-math-forked-tksxb?file=/src/components/HelloWorld.vue

I am wondering how to fix this issue.

**Comment by loveklmn** [comment, marked as answer]
solved when change to `nodeInputRule({find:regex, type:this.type})]`

**Reply by EverchronGreg** [reply]
Where does this code exist? I am having the same issue, but am not sure what code was changed to this, or in what file, if you could help please

**Reply by minskiter** [reply]
I am having the same issue. How to solve this issue following the answer?

**Reply by minskiter** [reply]
Oh~ I see. The behavior of `nodeInputRule` has a break change! `nodeInputRule(find,type,getAttributes)`->`nodeInputRule({find,type,getAttributes})`

**Reply by bdbch** [reply]
@minskiter this wasn't really a breaking change. Tiptap used this way to create nodeInputRules since around 9 months). I'll see if I can help you in your issue. I'll let you know!

**Reply by minskiter** [reply]
Thank you! I found where the error occurred. I used `prosemirror-inputrules` directly instead of `nodeInputRule`. I will close the issue.

**Reply by bdbch** [reply]
Does your example work fine now? I reopened your issue as I think Tiptap input rules should support `prosemirror-inputrules` and updated the title.

**Reply by EverchronGreg** [reply]
Do you know what combination of vue-2, starter-pack and tiptap core can be used to not display this error? Having trouble finding one. Or is it better to fork and make the above change, if that is what is being suggested?
